site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -207,6 +207,22 @@
         dropzone.addEventListener("dragleave", draghandle, false);
         dropzone.addEventListener("drop", dropWithMultipleZips, false);
     </script>
+    <script>
+        $(function() {
+            var paths = Object.keys($.parseJSON('{{ directory_priorities|json_encode|raw }}'));
+            for (let x = 0; x < paths.length; x++){
+                paths[x] = paths[x].slice(69);
```

Comment:
Why was 69 chosen for the maximum path length here?  The logic behind this decision should be documented in a comment for developers who may stumble upon this code in the future.  As it is now, this number seems rather arbitrary.

Reply:
If that is the case, a new approach needs to be taken here.  Course names don't necessarily have a consistent length.  A hacky way to do this would be to split on slashes, and then remove the first 8 components, but there is probably a more correct way to do this.

Reply:
Yeah, I hardcoded to 69, assuming the path will remain the same. So maybe when it sees course_materials, then cut out the rest of the path names, since course_materials should stay the same name?